schema_based_on_diagram.sql

```diff
@@ -1 +1,45 @@
+CREATE TABLE treatments (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    type VARCHAR(30),
+    name VARCHAR(30),
+    PRIMARY KEY(id),
+);
+
+CREATE TABLE medical_histories (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    admitted_at TIMESTAMP,
+    patient_id INT,
+    status VARCHAR(30),
+    PRIMARY KEY(id),
+    FOREIGN KEY(patient_id) REFERENCES patients(id)
+);
+
```

Comment:
- [ ] Good job creating the main tables for your project :+1:, but as a project requirement all "join tables" for the many-to-many relationships should be created, even if they are not shown on the diagram. Therefore you should create a table between medical_histories and treatments  (name can be different):

**Hint:**
```sql
  CREATE TABLE medical_histories_has_treatments (
    medical_history_id int refrences medical_histories(id),
    treatment_id int refrences treatments(id),
    );
```